samples/cellular/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -589,6 +755,38 @@ This is only supported on the `Nordic nRF9160 DK`_ with an attached nRF7002 EK.
 
 See also :ref:`the paragraphs on the Wi-Fi location tracking method <nrf_cloud_mqtt_multi_service_wifi_location_tracking>`.
 
+.. _nrf_cloud_mqtt_multi_service_building_wifi_conn:
+
+Building with experimental Wi-Fi connectivity support (for nRF5340 DK / nRF7002 EK) (demonstration only)
+========================================================================================================
+
+This sample experimentally supports connecting to nRF Cloud using Wi-Fi instead of using LTE.
+
+Presently, an overlay for this is only provided for the nRF5340 DK with nRF7002 EK shield attached is supported.
+It is possible to use Wi-Fi with other hardware combinations (such as the nRF7002 DK), but you must adjust heap and stack usage accordingly.
+See :file:`src/prj.conf` and :file:`overlay_nrf7002ek_wifi_no_lte.conf` for additional details.
+
+Connecting to nRF Cloud using Wi-Fi is currently only possible with using the experimental (and insecure) Kconfig option :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES`.
+See :ref:`nrf_cloud_mqtt_multi_service_hardcoded_certs` for details on setting it up.
+
+Once it is set up, you can configure your build to use Wi-Fi connectivity (on nRF5340 DK with nRF7002 EK shield) by using the ``--board nrf5340dk_nrf5340_cpuapp`` target and the ``-DSHIELD=nrf7002ek`` and ``-DOVERLAY_CONFIG=overlay_nrf7002ek_wifi_no_lte.conf`` options (in addition to the options required by :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES`).
```

Comment:
why not just add the required configs to the overlay?
can `overlay_nrf7002ek_wifi_no_lte.conf` be used without these added configs?

```
CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES=y 
CONFIG_NRF_CLOUD_CLIENT_ID_SRC_COMPILE_TIME=y 
CONFIG_NRF_CLOUD_CLIENT_ID=""
```

there is a build assert if client ID is empty.

Reply:
Mainly to discourage the user from viewing this as a permanent solution, I want setting up hardcoded certs to remain a manual step that the user must perform

Reply:
I understand, but currently, can overlay_nrf7002ek_wifi_no_lte.conf be used (functionally) without these added configs?

if it can't, then you are just making it more difficult for the user.

if it can, then yes, leave as is.

Reply:
> I understand, but currently, can overlay_nrf7002ek_wifi_no_lte.conf be used (functionally) without these added configs?

Essentially no. If an alternative to `CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` existed, then yes, these overlays could be used with that alternative instead. But otherwise, no.

> if it can't, then you are just making it more difficult for the user.

Yeah, that is the intent -- Since `CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` is for testing purposes only, I sincerely do not want to have it automatically enabled inside any overlays. I want its use to be a conscious decision by the user.